LayoutTests/svg/text/font-small-enlarged-minimum-larger-expected.svg

```diff
@@ -1,7 +1,4 @@
 <svg xmlns="http://www.w3.org/2000/svg" style="width: 400px; height: 400px;">
-  <script>
-    internals.settings.setMinimumFontSize(80.0);
-  </script>
   <rect x="40" y="40" width="320" height="320" stroke="green" fill="none" stroke-width="2"/>
-  <text x="200" y="200" font-size="40.0" fill="black">1</text>
+  <text x="200" y="200" font-size="80.0" fill="black">1</text>
```

Comment:
Why is change related to this patch? Does not setMinimumFontSize() above set the minimum font size in the SVG document be 80?